pysmt/factory.py

```diff
@@ -261,6 +261,16 @@ def is_sat(self, formula, quantified=None, solver_name=None, logic=None):
         solver = self.Solver(quantified=quantified, name=solver_name, logic=logic)
         return solver.is_sat(formula)
 
+    def get_model(self, formula, quantified=None, solver_name=None, logic=None):
+        solver = self.Solver(quantified=quantified, name=solver_name, logic=logic)
+        solver.add_assertion(formula)
+        check = solver.solve()
+        retval = None
+        if check:
+            retval = solver.get_model()
+        solver.exit()
```

Comment:
Good catch! In all other shortcuts we are not destroying the solver!!! (e.g., see is_valid below). Maybe it would be better to use a context in all of these.